### ISSUE TITLE
Title: PermissionError and Path Issues for rfp task

### ISSUE THREAD

**Issue by yassineCh** [lamda-bbo/offline-moo#1]
Thank you for this project. 

I encountered the following PermissionError when running the RFP task:
off_moo_bench/problem/lambo/lambo/tasks/proxy_rfp/foldx.py", line 93, in init
os.makedirs(self.work_dir, exist_ok=True)
PermissionError: [Errno 13] Permission denied: '/home/xk'

The path causing the issue: PosixPath('/home/xk/trx/Offline-MOO/m2bo_bench/problem/lambo/data/experiments/test/sour_plank_0/2024-05-22_11-08-56/foldx')

I manually changed it to my home directory and updated `m2bo_bench` to `off_moo_bench`, but then faced a missing file. FileNotFoundError:
off_moo_bench/problem/lambo/data/experiments/test/sour_plank_0/2024-05-22_11-08-56/foldx/23dd24d87b91449b9ed4386f1506860d/wt_input_Repair_1.pdb'

Please provide guidance on how to configure the project to use the appropriate directories?

**Comment by trxcc** [comment]
Hi @yassineCh 

Thanks for your issue!

After checking our source code, I found bugs in 
generating ``/off_moo_bench/problem/lambo/data/experiments/test/proxy_rfp_problem.pkl`` instance. 

Please generate a new problem instance by running
```shell
cd off_moo_bench/problem/lambo/
python scripts/black_box_opt.py optimizer=mf_genetic optimizer/algorithm=nsga2 task=proxy_rfp tokenizer=protein
```

I have checked that the results evaluated by the newly generated instance are the same as that which are evaluated by the instance we provided.

We have fixed these bugs in our repository, and added these fixes to our ``install.sh`` and ``README.md``. 

If you have any questions, feel free to contact us.

**Comment by yassineCh** [comment]
Thank you for the fix. The issues are resolved now. Please change  
```bash 
cd m2bo_new 
``` 
in `install.sh`

**Comment by trxcc** [comment]
Thank you for your suggestion. We have fixed it in ``install.sh``. 

Besides, we have checked the installation script again, and fixed some bugs regarding ``MuJoCo`` and ``FoldX``. We suggest that you use a machine with ``CUDA version 11.7`` or higher. We have tested more different hardware environments, including:
- Ubuntu 22.04, 1x3090, CUDA 12.0
- Ubuntu 20.04, 2x3090, CUDA 11.8
- Ubuntu 20.04, 2xA6000, CUDA 11.8

and we successfully installed it. If you have any issues, feel free to contact us. 

We are working on a more formal and stable version of baselines. Thank you.
